scripts/windows/install-wsl.ps1

```diff
@@ -0,0 +1,48 @@
+# //////////////////////////////////////////////////////////////////////
+# install-wsl.ps1
+# See https://docs.microsoft.com/en-us/windows/wsl/install-win10 for the official story behind this code
+
+Param([ValidateSet("EnableWSL-01", "EnableVMPlatform-02", "InstallLinuxUpdatePackage-03")]$Step = "EnableWSL-01")
+
+$workDir = (Join-Path ([System.IO.Path]::GetTempPath()) rdinstall)
+New-Item -ItemType Directory -Force -Path $workDir
+
+$logFile = (Join-Path $workDir restarts.txt)
+$wslMsiFile = (Join-Path $workDir wsl_update_x64.msi)
+
+$script = $myInvocation.MyCommand.Definition
+$scriptPath = Split-Path -parent $script
+. (Join-Path $scriptpath restart-helpers.ps1)
+$sudoInstallScript = (Join-Path $scriptPath sudo-install-wsl.ps1)
+
+#Requires -RunAsAdministrator
```

Comment:
Might be a good idea to say that this is not a comment but a directive… because PowerShell.

Reply:
done, used the same comment as in the other install script.